woocommerce/woocommerce-android
# Remove Stats V3 Code

Summary
==========
Fixes issue #2886 and #2068. Since now only the Stats V4 API is supported through the My Store group of views, this PR is a major code cut removing everything related to the old Dashboard group of views with the Stats V3 API integration.

Since this is mostly a code removal, the main point of this PR is to make sure that all Stats V4 still works as it should and prompts errors when the user login with a Stats V3 site.

How to Test
==========
Scenario 1
-----------

1. Login with a valid Stats V4 site
2. Verify that both Revenue and Top Performers fields are working properly as expected
3. Verify that the Top Performers offline features are working as expected
4. Verify that clicking any Top Performers properly opens the Product Details view

Scenario 2
-----------

1. Login with a Stats V3 site
2. Verify that a view is loaded prompting that `We can't display your store's analytics`

Screenshots
==========
| Light Mode  | Dark Mode |
| ------------- | ------------- |
| ![screenshot-1603868577241](https://user-images.githubusercontent.com/5920403/97403152-a7a8fa00-18d2-11eb-9323-1dad08d2567b.jpg) | ![screenshot-1603868556781](https://user-images.githubusercontent.com/5920403/97403154-aa0b5400-18d2-11eb-8b5b-99b3d37ff928.jpg) |

Update release notes:
